README.md

```diff
@@ -405,21 +405,19 @@ await octokit.rest.issues.create({
 });
 ```
 
-The REST API endpoint methods are generated automatically from [GitHub's OpenAPI specification](https://github.com/github/rest-api-description/). We track operation ID or parameter name changes in order to implement deprecation warnings and reduce the frequency of breaking changes.
+The `octokit.rest` endpoint methods are generated automatically from [GitHub's OpenAPI specification](https://github.com/github/rest-api-description/). We track operation ID and parameter name changes in order to implement deprecation warnings and reduce the frequency of breaking changes.
 
-Every endpoint method is an instance of `octokit.request` with defaults set, so it supports the same parameters as well as the `.endpoint()` API.
+Under the covers, every endpoint method is actually an instance of `octokit.request` with defaults set, so it supports the same parameters as well as the `.endpoint()` API.
 
 #### Arbitrary requests
 
-**standalone method**: [`@octokit/request`](https://github.com/octokit/request.js#readme)
-
-An http request can be sent using `octokit.request`. It's API is matching GitHub's REST API documentation 1:1.
+You can call the GitHub REST API directly using `octokit.request`. The `reqeust` API matches GitHub's REST API documentation 1:1 so anything you see there, you can call using `request`. See [`@octokit/request`](https://github.com/octokit/request.js#readme) for all the details.
```

Comment:
If `rest` matches the schema and `request` matches the doc and the doc is driven from the schema, shouldn't they all match?  I get that there may be point in time gaps but in theory the team is looking to drive that window to (near) zero and have the schema be the authority for everything. Given that, how should we position `request`?

Perhaps focusing on "you can cut and past the doc's api URL path directly"?